src/main/java/com/ge/predix/log4j1/PredixLayoutPattern.java

```diff
@@ -30,10 +30,12 @@ public PredixLayoutPattern() {
     protected String convert(final LoggingEvent event) {
         // need LinkedHashMap to preserve order of log fields
         Map<String, Object> logFormat = new LinkedHashMap<>();
+        Object appName = event.getMDC("APP_NAME") != null ? event.getMDC("APP_NAME") : event.getMDC("VCAP_APP_NAME");
```

Comment:
can we decide between the two in the filter at request level to avoid doing this for every log event? So use only one key in MDC, set to the preferred name.